src/components/MySongsTable/MySongsTable.js

```diff
@@ -0,0 +1,15 @@
+import mySongsTableTemplate from "./MySongsTable.html";
+export class MySongsTableComponent {
+  constructor(mountPoint, props = {}) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  mount() {
+    this.mountPoint.innerHTML = this.render();
+  }
+
+  render() {
+    return mySongsTableTemplate(this.props);
```

Comment:
please specify what exactly from props you are passing to the template

Reply:
resolved. specified props